hayeon/programmers-1/콜라-문제.js

```diff
@@ -0,0 +1,20 @@
+function solution(a, b, n) {
+  let result = 0;
+
+  while (n / a >= 1) {
+    let newCokes = Math.floor(n / a) * b;
+
+    results += newCokes;
+
+    n = (n % a) + newCokes;
+  }
+  return result;
+}
```

Comment:
```suggestion
function solution(a, b, n) {
  let emptyBottleCount = n;
  let result = 0;

  while (emptyBottleCount / a >= 1) {
    const newCokes = Math.floor(emptyBottleCount / a) * b;

    results += newCokes;

    emptyBottleCount = (emptyBottleCount % a) + newCokes;
  }
  return result;
}
```
- n을 특정 변수에 넣어주는건 어떨까요!? 더 명확한 의미를 부여하면 좋을 것 같습니다.
- while문에서 newCokes를 const로 해도 될 것 같아요~!

Reply:
헉 const로 해도 되는군요!!! 감사합니다! 변수명은 아직도 어렵지만.. 노력해보겠습니다!